default.nix

```diff
@@ -22,7 +22,7 @@ let
 in
 stdenv.mkDerivation {
   pname = "nixd";
-  version = "0.0.1";
+  version = "nightly";
```

Comment:
Can we get `git describe` in the package script? @Ruixi-rebirth

Reply:
LGTM